static_test_driver/static_test_driver.ino

```diff
@@ -19,80 +19,74 @@
 #include <avr/pgmspace.h>
 
 // LEDs
-#define THERMO1_LED 23
-#define THERMO2_LED 25
-#define THERMO3_LED 27
-#define FORCE_LED 29
-#define PRESSURE_LED 31
-#define STATE_LED 33
-#define STATUS_LED 35
+#define THERMO1_LED 28
+#define THERMO2_LED 30
+#define THERMO3_LED 32
+#define FORCE_LED 42
+#define PRESSURE_FUEL_LED 44
+#define PRESSURE_OX_LED 50 
+#define INLET_TEMP_LED 46 
+#define OUTLET_TEMP_LED 48 
+#define STATE_LED 38 //TODO: Discard STATE_LED code for current sensors revision due to pin conflict. Pin # was changed to satisfy next revision
+#define STATUS_LED 40 //TODO: Discare STATUS_LED code for current sensors revision due to pin conflict. Pin # was changed to satisfy next revision
 
 // Accelerometer
 Adafruit_MMA8451 mma;
 
-// Thermocouple Setup
-#define MAXDO   24
-#define MAXCLK  22
-
-#define MAXCS1  43
-#define MAXCS2  30
-Adafruit_MAX31855 inlet_thermocouple(MAXCLK, MAXCS1, MAXDO);
-Adafruit_MAX31855 outlet_thermocouple(MAXCLK, MAXCS2, MAXDO);
-
-// Thermocouple and pressure setup for MK_2
-#if CONFIGURATION == MK_2
-#define MAXCS3  32
-Adafruit_MAX31855 chamber_thermocouple(MAXCLK, MAXCS3, MAXDO);
+//Analog Temperature Setup
+#define INLET_TEMP A4
+#define OUTLET_TEMP A5
 
+//Pressure Setup
 #define PRESSURE_CALIBRATION_FACTOR 246.58
 #define PRESSURE_OFFSET 118.33
-#define PRESSURE_PIN A1
-
+#define PRESSURE_FUEL A2
+#define PRESSURE_OX A3 
 #define PRESSURE_NUM_HIST_VALS 10
 
-float pressure_hist_vals[PRESSURE_NUM_HIST_VALS];
-int pressure_val_num = 0;
-bool pressure_zero_ready = false;
-float pressure_zero_val = 0;
+float pressure_hist_vals_1[PRESSURE_NUM_HIST_VALS];
+int pressure_val_num_1 = 0;
+bool pressure_zero_ready_1 = false;
+float pressure_zero_val_1 = 0;
+
+float pressure_hist_vals_2[PRESSURE_NUM_HIST_VALS];
+int pressure_val_num_2 = 0;
+bool pressure_zero_ready_2 = false;
+float pressure_zero_val_2 = 0;
+
+// Thermocouple setup for MK_2
+#if CONFIGURATION == MK_2
+#define MAXDO   24
+#define MAXCLK  52
+#define MAXCS1  22
+#define MAXCS2  33
+#define MAXCS3  31
+Adafruit_MAX31855 chamber_thermocouple_1(MAXCLK, MAXCS1, MAXDO);
+Adafruit_MAX31855 chamber_thermocouple_2(MAXCLK, MAXCS2, MAXDO);
+Adafruit_MAX31855 chamber_thermocouple_3(MAXCLK, MAXCS3, MAXDO);
 #endif
 
 // Load cell setup
+#if CONFIGURATION == MK_2
 #define LOAD_CELL_CALIBRATION_FACTOR 20400.0 //This value is obtained using the SparkFun_HX711_Calibration sketch
-#define LOAD_CELL_DOUT 32
-#define LOAD_CELL_CLK  26
-HX711 scale(LOAD_CELL_DOUT, LOAD_CELL_CLK);
-
-// Force plate setup
-#define FORCE_PLATE_PIN A2
```

Comment:
How absolutely sure are we that we will never use the force plate again for any purpose (e.g. demos like at the STEM fair last year?)  If so we should at least leave in the force plate code, but disable it via the preprocessor (e.g. make it part of the `DEMO` mode?)  If we know we will never use it again for sure though then it is fine to take it out.

Reply:
OK, sounds good then.